Automattic/woocommerce-payments
# Format files using prettier rules

This PR is strictly to prepare for merging #631.

This also adds prettier ignores to eslint and stylelint so the linters won't go bonkers during CI. This includes adding the necessary npm packages to the package.json and package-lock.json.

#### Changes proposed in this Pull Request

* Format the whole repository according to the formatting rules used by wp-prettier in #631.

#### Testing instructions



* Install the new packages necessary for the linters with `npm i`
* Make sure linting passes with `npm run lint` (or just make sure CI passes)

-------------------
